packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/LinkedPaymentAccounts/Accounts/index.tsx

```diff
@@ -182,8 +200,8 @@ class Payments extends PureComponent<InjectedFormProps<{}, Props> & Props> {
       value
     }))
 
-    const defaultCardMethod = this.props.paymentMethods.methods.find(
-      m => m.type === 'PAYMENT_CARD' && orderType === 'BUY'
+    const bankTransfer = defaultMethods.find(
+      method => method.value.type === 'BANK_TRANSFER' && orderType === 'BUY'
     )
```

Comment:
It's before this change but I don't understand why `orderType === 'BUY'` is inside the `find`. It's either true or false for all items in the list.